acts_as_paranoid.gemspec

```diff
@@ -20,8 +20,8 @@ Gem::Specification.new do |spec|
   spec.files = File.read("Manifest.txt").split
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "activerecord", ">= 5.2", "< 7.0"
-  spec.add_runtime_dependency "activesupport", ">= 5.2", "< 7.0"
+  spec.add_runtime_dependency "activerecord", ">= 5.2", "<= 7.0"
+  spec.add_runtime_dependency "activesupport", ">= 5.2", "<= 7.0"
```

Comment:
Please use "< 7.1" instead, so patch updates to Rails will be allowed.

Reply:
okay sure